app/models/game.rb

```diff
@@ -36,4 +36,27 @@ def other_player(player)
   def pass_turn(player)
     update_attributes(active_player_id: other_player(player))
   end
+
+  def stalemate?
+    king_not_in_check? && available_move?
+  end
+
+  def king_not_in_check?
+    king = pieces.find_by(type: 'King', color: current_color)
+    return true unless king.in_check?
+    false
```

Comment:
could just have `!king.in_check?` but what you have works perfectly well, just my two cents.